apps/authentication-service/src/resources/clients/clients.test.ts

```diff
@@ -0,0 +1,36 @@
+import { Client } from "auth0";
+import {
+  getClient,
+  createClient,
+  deleteClient,
+} from "@/resources/clients/clientsService";
+
+describe("Clients test", () => {
+  const clientId: string = "ucp-test-client";
+  const clientName = "Ucp Test Client";
+  const clientDesc = "For unit testing";
+
+  it("Test Client Creation", async () => {
```

Comment:
These it statements aren't describing to me what the test is doing.

it("returns a new client")